src/Microsoft.ML.Data/Evaluators/BinaryClassifierEvaluator.cs

```diff
@@ -789,6 +789,34 @@ private void ComputePrCurves()
             }
         }
 
+        /// <summary>
+        /// Generates precision recall curve data points.
+        /// </summary>
+        /// <param name="data">The scored data.</param>
+        /// <param name="label">The name of the label column in <paramref name="data"/>.</param>
+        /// <param name="score">The name of the score column in <paramref name="data"/>.</param>
+        /// <param name="probability">The name of the probability column in <paramref name="data"/>, the calibrated version of <paramref name="score"/>.</param>
+        /// <param name="predictedLabel">The name of the predicted label column in <paramref name="data"/>.</param>
+        /// <returns>The precision recall curve data points.</returns>
+        public IDataView GetPrecisionRecallCurve(IDataView data, string label, string score, string probability, string predictedLabel)
```

Comment:
Unit tests & sample is tbd

Reply:
sample is done, tests tbd

---
In reply to: [267591483](https://github.com/dotnet/machinelearning/pull/3039#discussion_r267591483) [](ancestors = 267591483)

Reply:
test is added

---
In reply to: [268037217](https://github.com/dotnet/machinelearning/pull/3039#discussion_r268037217) [](ancestors = 268037217,267591483)